File: micronaut-mcp-server-java-sdk/src/test/java/io/micronaut/mcp/server/stateless/sync/tools/jsonschema/FenEvaluationRequest.java
package io.micronaut.mcp.server.stateless.sync.tools.jsonschema;

import io.micronaut.core.annotation.Introspected;
import io.micronaut.jsonschema.JsonSchema;

/**
 *
 * @param fen A Chess position in Forsyth–Edwards Notation
 */
@JsonSchema
@Introspected
public record FenEvaluationRequest(String fen) {
}

File: test-suite-docs-python/src/test/python/micronaut/mcp/docs/tools/jsonschema/FenEvaluationRequest.py
# tag::imports[]
from dataclasses import dataclass

from micronaut.jsonschema import JsonSchema
from micronaut.serde.annotation import Serdeable
# end::imports[]


# tag::clazz[]
@JsonSchema
@Serdeable
@dataclass
class FenEvaluationRequest:
    fen: str
    """A Chess position in Forsyth–Edwards Notation"""
# end::clazz[]
